### ISSUE TITLE
Issue with "Array.prototype.forEach()": …

### ISSUE THREAD

**Issue by chrisdavidmills** [mdn/content#51]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach

  #### What information was incorrect, unhelpful, or incomplete?

  #### Specific section or headline?

  #### What did you expect to see?

  #### Did you test this? If so, how?

  
  <details>
  <summary>MDN Content page report details</summary>

  * Folder: `en-us/web/javascript/reference/global_objects/array/foreach`
  * MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
  * GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/array/foreach/index.html
  * Report started: 2020-10-13T14:10:03.950Z

  </details>

**Comment by peterbe** [comment]
Well that didn't take long for some git to not bother replacing the `...` with an explanation for what the real problem is :)

Seriously though, what would you have put into the title there? Is it annoyingly redundant to have to express what the problem is once in the title and then once under the "What information was incorrect, unhelpful, or incomplete?" question. 

Another thought I had was; should one of the questions be more proactive towards solving the problem? I.e. something like "Do you have a suggestion to fix the issue?" or "Any immediate ideas how to resolve this issue?"
Basically, anything to "turn it around" on the reporter so they A) understand that they can be part of the solution, B) they might already know more than anybody on the topic so we should capitalize on it.
